### ISSUE TITLE
Training the fine-tuned base line with standard supervised learning with union/concatenation of labels

### ISSUE THREAD

**Issue by brando90** [mboudiaf/pytorch-meta-dataset#18]
Hi @mboudiaf, I wanted to train the fine-tuned baseline from meta-data set (MDS) i.e. concatenate/union all the data sets and all the labels and then train in normal supervised learning. Is the right way to do this this:

https://github.com/mboudiaf/pytorch-meta-dataset/blob/c6d6922003380342ab2e3509425d96307aa925c5/example.py#L173

I am mainly asking because there needs to be some sort of relabling that takes into account all the data set labels and wanted to know how that was done.

Thank you!
